toonz/sources/tnzbase/CMakeLists.txt

```diff
@@ -207,17 +207,17 @@ elseif(APPLE)
     )
 
     target_link_libraries(tnzbase Qt5::Core Qt5::Gui)
-elseif(UNIX)
+elseif(BUILD_ENV_GCC)
     _find_toonz_library(EXTRA_LIBS "tnzcore")
-    set(EXTRA_LIBS ${EXTRA_LIBS}
-        ${SDL_LIB_LIBRARIES}
-    )
-
-    include_directories(
-        SYSTEM
-        ${SDL_LIB_INCLUDE_DIRS}
-    )
-    target_link_libraries(tnzbase Qt5::Core Qt5::Gui)
```

Comment:
You right, this line got here by mistake. I wanted to ask you about two lines [209](/opentoonz/opentoonz/blob/master/toonz/sources/tnzbase/CMakeLists.txt#L209) and [220](/opentoonz/opentoonz/blob/master/toonz/sources/tnzbase/CMakeLists.txt#L220)(this line) in this file which seems unnecessary, because command in final line replaces them:

```
if(WIN32)
    set(EXTRA_LIBS
        tnzcore
    )
elseif(APPLE)
    find_library(TWAIN_LIB TWAIN)
    find_library(IOKIT_LIB IOKit)
    find_library(COCOA_LIB Cocoa)

    set(EXTRA_LIBS
        ${CMAKE_CURRENT_BINARY_DIR}/../tnzcore/libtnzcore.dylib
        ${USB_LIB}
        ${TWAIN_LIB}
        ${IOKIT_LIB}
        ${COCOA_LIB}
    )

    target_link_libraries(tnzbase Qt5::Core Qt5::Gui) # <------------------ this
elseif(UNIX)
    _find_toonz_library(EXTRA_LIBS "tnzcore")
    set(EXTRA_LIBS ${EXTRA_LIBS}
        ${SDL_LIB_LIBRARIES}
    )

    include_directories(
        SYSTEM
        ${SDL_LIB_INCLUDE_DIRS}
    )
    target_link_libraries(tnzbase Qt5::Core Qt5::Gui) # <------------------ and this
endif()

target_link_libraries(tnzbase Qt5::Core Qt5::Gui ${EXTRA_LIBS}) # <-------- and finally
```

I'll fix this commit. Will it right to remove both these lines, or them have some meaning?